### PR TITLE
Update auth.rst

### PR DESCRIPTION
**Pull request by aavrug**
This also can be use in place of the above one line. It is easy to understand  because in previous we used patchEntity and now surprisingly patchEntity has been removed..

### CODE REVIEW

File: en/tutorials-and-examples/blog-auth-example/auth.rst [hunk 1]
```diff
@@ -301,6 +301,9 @@ logged in user as a reference for the created article::
         if ($this->request->is('post')) {
             // Added this line
             $article->user_id = $this->Auth->user('id');
+            // Also can be used the two lines
+            //$newData = ['user_id' => $this->Auth->user('id')];
+			//$post = $this->Posts->patchEntity($post, $newData);
```
Comment (antograssiot) [inline en/tutorials-and-examples/blog-auth-example/auth.rst]: in the example, table name seems to be articles not posts

**Comment by aavrug** [comment]
I have changed it. Thanks for correcting me.

File: en/tutorials-and-examples/blog-auth-example/auth.rst [hunk 2]
```diff
@@ -301,6 +301,9 @@ logged in user as a reference for the created article::
         if ($this->request->is('post')) {
             // Added this line
             $article->user_id = $this->Auth->user('id');
+            // Also can be used the two lines
+            //$newData = ['user_id' => $this->Auth->user('id')];
+	    //$article = $this->Articles->patchEntity($article, $newData);
```
Comment (markstory) [inline en/tutorials-and-examples/blog-auth-example/auth.rst]: Indentation looks inconsistent and the phrase might be better as 'You could also do the following'

**Comment by antograssiot** [comment]
thank's @aavrug

**Pull request merged**